### ISSUE TITLE
Logi Bolt USB Receiver

### ISSUE THREAD

**Issue by ikarus23** [Logitech/fw_updates#37]
Since version 2.0 of ScriptDFU, firmware updates for Bolt RCV and Bolt RCV paired devices should be supported. I want to test that. However, all the firmware in this repository is close to 2 years old and I can not find firmware for Logi Bolt devices.

```
C:\Users\test\Downloads> .\logitech_dfu_2.0_win.exe /list
1     Logi Bolt receiver [user-action]
1.1   MX Keys B [off-line]
PS C:\Users\test\Downloads> .\logitech_dfu_2.0_win.exe /available
1
1.1
PS C:\Users\test\Downloads> .\logitech_dfu_2.0_win.exe /version
Logitech Device Updater 2.0.170008 (Windows build)

Copyright (c) 2021 Logitech
```
Any hints?

**Comment by bstanzli** [comment]
same question here, seems that this is the only tools that is able to update firmware on Logi for business devices but firmware are nowhere to be found yet..

**Comment by fanoush** [comment]
I guess you already know but still - here are release notes for the firmware https://support.logi.com/hc/en-us/articles/13629864225303-Logi-Bolt-Receiver-Firmware-Release-Notes but sadly with no download.  From the "SoftDevice update" message I guess these dongles are likely based on nrf52840. Anyone took one apart? Or even has link to firmware files?

**Comment by ikarus23** [comment]
Yes, i took one apart, but I misplaced it somewhere. Don't remember the exact version, but yes, it was some sort of nRF52840. The MX Keys for business keyboard has an nRF52832.

Since there are firmware updates available now, you should be able to extract them from the normal update tool. I did this for other Logi firmware in the past. Not sure if I will find the time to do it soon.

**Comment by fanoush** [comment]
>  The MX Keys for business keyboard has an nRF52832.

also the Signature K650 has it, sadly it is the QFAB 32KB RAM/256KB flash variant,  got one from 'to recycle' bin in the office and took it apart https://ibb.co/album/MDzC18

> Since there are firmware updates available now, you should be able to extract them from the normal update tool. 

Thanks, I don't know what 'normal update tool' is and currently even don't have the Bolt dongle but was thinking about getting one if there is indeed nrf52840 inside (or possibly nRF52833, that one has USB interface too)
